src/bin/itm-decode.rs

```diff
@@ -0,0 +1,37 @@
+use itm::{Decoder, DecoderError, DecoderOptions};
+use std::fs::File;
+use std::path::PathBuf;
+use structopt::StructOpt;
+
+#[derive(StructOpt, Debug)]
+#[structopt(
+    about = "An ITM/DWT packet protocol decoder, as specified in the ARMv7-M architecture reference manual, Appendix D4. See <https://developer.arm.com/documentation/ddi0403/ed/>. Report bugs and request features at <https://github.com/rust-embedded/itm>."
+)]
+struct Opt {
+    #[structopt(long = "--ignore-eof")]
+    ignore_eof: bool,
+
+    #[structopt(name = "FILE", parse(from_os_str), help = "Raw trace input file.")]
+    file: PathBuf,
+}
```

Comment:
An option for `TimestampedTracePackets` should be added. `itm-decode` will be a thin wrapper around the library.

Reply:
A `--timestamps` option (with some other auxilliary flags) have been added.